From 7c65f2093a6c0fc3f2ee8af360a518a3fe92fa3f Mon Sep 17 00:00:00 2001
From: Swati Agarwal <quic_swatagar@quicinc.com>
Date: Thu, 24 Jul 2025 13:01:40 +0530
Subject: [PATCH] PENDING: arm64: dts: qcom: Enable usb0 support on QCS8275 IQ8
 EVK

Enable primary controller usb0 on QCS8275 IQ8 EVK for adb support.

Signed-off-by: Swati Agarwal <quic_swatagar@quicinc.com>
Upstream-Status: Pending
---
 .../boot/dts/qcom/qcs8275-iq-8275-evk.dts     | 24 +++++++++++++++++++
 1 file changed, 24 insertions(+)

diff --git a/arch/arm64/boot/dts/qcom/qcs8275-iq-8275-evk.dts b/arch/arm64/boot/dts/qcom/qcs8275-iq-8275-evk.dts
index 838e52d0b670..c997637ee032 100644
--- a/arch/arm64/boot/dts/qcom/qcs8275-iq-8275-evk.dts
+++ b/arch/arm64/boot/dts/qcom/qcs8275-iq-8275-evk.dts
@@ -224,3 +224,27 @@ dp_hot_plug_det: dp-hot-plug-det-state {
 &uart7 {
 	status = "okay";
 };
+
+&usb_1 {
+	status = "okay";
+};
+
+&usb_1_dwc3 {
+	usb-role-switch;
+	dr_mode = "peripheral";
+};
+
+&usb_1_hsphy {
+	vdda-pll-supply = <&vreg_l7a>;
+	vdda18-supply = <&vreg_l7c>;
+	vdda33-supply = <&vreg_l9a>;
+
+	status = "okay";
+};
+
+&usb_qmpphy {
+	vdda-phy-supply = <&vreg_l7a>;
+	vdda-pll-supply = <&vreg_l5a>;
+
+	status = "okay";
+};
-- 
2.34.1

